Framework/PythonInterface/plugins/algorithms/SNSPowderReduction.py

```diff
@@ -215,8 +216,15 @@ def PyInit(self):
                              "List of all output file types. Allowed values are 'fullprof', 'gsas', 'nexus', "
                              "'pdfgetn', and 'topas'")
         self.declareProperty("OutputFilePrefix", "", "Overrides the default filename for the output file (Optional).")
-        self.declareProperty(FileProperty(name="OutputDirectory",defaultValue="",action=FileAction.Directory))
-        self.copyProperties('AlignAndFocusPowderFromFiles', 'CacheDir')
+        self.declareProperty(FileProperty(name="OutputDirectory", defaultValue="",action=FileAction.Directory))
+
+        # Optimization options
+        self.declareProperty(FileProperty(name='CacheDir', defaultValue='', action=FileAction.Directory))
```

Comment:
Is it possible for us to use the original `copyProperties` for `CacheDir`? 
Or perhaps change the CacheDir def in AlignAndFocuse?

Reply:
Fixed. AlignAndFocuse actually copy `CacheDir` from `CleanFileCache`, so I updated property `CacheDir` ub `CleanFileCache` to be a directory browse dialog